spec/integration/groups_index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+# rspec for features
+RSpec.describe 'groups#index', type: :feature do
+  let!(:user) { User.create(name: 'Akai', email: 'akaiisred@gmail.com', password: '123123') }
+  let!(:group) { Group.create(name: 'Medicine', user_id: user.id) }
+  let!(:expense) { Expense.create(name: 'Gerdex', amount: 5, author_id: user.id, group_ids: ['', group.id]) }
+  let!(:expense2) { Expense.create(name: 'Acetaminofen', amount: 7, author_id: user.id, group_ids: ['', group.id]) }
+
+  before :each do
+    user.confirm
+    sign_in user
```

Comment:
- [ ] There are 3 test cases failing, Please make sure that all test cases are passing and green.

![image](https://github.com/Swarzstein/ror-capstone/assets/68796232/9eea4d4a-8ba1-4bac-af77-143c05ff54d6)